tests/python/relay/opencl_texture/test_conv2d_nchw_texture.py

```diff
@@ -577,7 +577,6 @@ def test_residual_block(target, dtype):
         "global.texture-weight",
         "global.texture",
         "global.texture-weight",
-        "global",
```

Comment:
there are two different graphs produced for fp16 and fp32. the correct fix should be like
```
index 504a2b4e3..8f1c9e1ab 100644
--- a/tests/python/relay/opencl_texture/test_conv2d_nchw_texture.py
+++ b/tests/python/relay/opencl_texture/test_conv2d_nchw_texture.py
@@ -570,19 +570,33 @@ def test_residual_block(target, dtype):
         "weight3": tvm.nd.array(filter_data3),
     }
-    static_memory_scope = [
-        "global",
-        "global.texture",
-        "global.texture-weight",
-        "global.texture-weight",
-        "global.texture",
-        "global.texture-weight",
-        "global",
-        "global.texture",
-        "global.texture-weight",
-        "",
-        "",
-    ]
+    if dtype == "float16":
+        static_memory_scope = [
+            "global",
+            "global.texture",
+            "global.texture-weight",
+            "global.texture-weight",
+            "global.texture",
+            "global.texture-weight",
+            "global",
+            "global.texture",
+            "global.texture-weight",
+            "",
+            "",
+        ]
+    else:
+        static_memory_scope = [
+            "global",
+            "global.texture",
+            "global.texture-weight",
+            "global.texture-weight",
+            "global.texture",
+            "global.texture-weight",
+            "global.texture",
+            "global.texture-weight",
+            "",
+            "",
+        ]
     build_run_compare(mod, params1, {"data": input_shape}, dtype, target, static_memory_scope)
```